absl/log/BUILD.bazel

```diff
@@ -57,7 +57,6 @@ cc_library(
 )
 
 cc_library(
-    alwayslink = True,
```

Comment:
Is this an intended change?

Reply:
Yes, alwayslink was added a couple lines below this line in the upstream repo.